runtime/i3c/src/core.rs

```diff
@@ -273,18 +273,10 @@ impl<'a, A: Alarm<'a>> I3CCore<'a, A> {
             }
```

Comment:
Is `interrupt-status` register a regular type or `write1toclear`?  If it is `write1toclear`, I think we should change this piece of code accordingly.

Reply:
Ah, you're correct, the docs say "woclr".